hack/jenkins/e2e-runner.sh

```diff
@@ -212,13 +253,23 @@ fi
 # # Move the permissions for the keys to Jenkins.
 # $ sudo chown -R jenkins /var/lib/jenkins/gce_keys/
 # $ sudo chgrp -R jenkins /var/lib/jenkins/gce_keys/
-if [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
-    echo "Skipping SSH key copying for AWS"
-else
-    mkdir -p ${WORKSPACE}/.ssh/
-    cp /var/lib/jenkins/gce_keys/google_compute_engine ${WORKSPACE}/.ssh/
-    cp /var/lib/jenkins/gce_keys/google_compute_engine.pub ${WORKSPACE}/.ssh/
-fi
+case "${KUBERNETES_PROVIDER}" in
+    gce|gke)
```

Comment:
could get away with putting `kubemark` in here, not that it's being exercised atm, but keeps parity with https://github.com/kubernetes/kubernetes/blob/75b01662e61d8cd2b63b0ceaa8631a00ac01fdc0/test/e2e/util.go#L2983